app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= fields_for item do |f| -%>
+  <% loaders = @operatingsystem.available_loaders; if loaders.any? -%>
+    <%= selectable_f f, :pxe_loader,
+      loaders.to_a, {
+        :include_blank => blank_or_inherit_f(f, :pxe_loader),
+      },
+      :label => _('PXE Loader'),
+      :help_inline => popover(_("DHCP filename option"), _("DHCP filename option to use. Set to None for non-PXE method (e.g. iPXE)"), :data => { :placement => 'top' }) %>
```

Comment:
Follow Patternfly's field level help here - http://www.patternfly.org/pattern-library/forms-and-controls/field-level-help/#_overview - we ought to standardize on that at some point, so the less changes we have to make then, the better.